### PR TITLE
Dump persistent cache options

### PR DESCRIPTION
**Pull request by yiwu-arbug**
Summary:
Dump persistent cache options

Test Plan:
Sample output:
https://gist.github.com/yiwu-arbug/b54822499fdaeda5ce770bfdf1698976

### CODE REVIEW

File: utilities/persistent_cache/block_cache_tier.cc [hunk 1]
```diff
@@ -28,6 +28,7 @@ Status BlockCacheTier::Open() {
 
   // Check the validity of the options
   status = opt_.ValidateSettings();
+  printf("status %s\n", status.ToString().c_str());
```
Comment (IslamAbdelRahman) [inline utilities/persistent_cache/block_cache_tier.cc]: Is this intentional ?

File: table/block_based_table_factory.cc [hunk 2]
```diff
@@ -139,6 +139,11 @@ std::string BlockBasedTableFactory::GetPrintableTableOptions() const {
              table_options_.block_cache->GetCapacity());
     ret.append(buffer);
   }
+  if (table_options_.persistent_cache) {
```
Comment (IslamAbdelRahman) [inline table/block_based_table_factory.cc]: We should print `nullptr` if it's not set

Comment (yiwu-arbug) [inline table/block_based_table_factory.cc]: I just following how we print `table_options_.block_cache`.

Comment (IslamAbdelRahman) [inline table/block_based_table_factory.cc]: actually we do print if `table_options_.block_cache` is `nullptr` in line 134
can we do the same here ?

Comment (yiwu-arbug) [inline table/block_based_table_factory.cc]: I see.

**Comment by siying** [comment]
Can you dump the path and size of the persistent cache too?

**Comment by siying** [comment]
Ignore my previous comment. I realized that it is already there. Good job!

**Pull request closed without merging**